code/globalincs/utility.h

```diff
@@ -147,4 +147,145 @@ typename T::size_type stringcost(const T& op, const T& input, typename T::size_t
     return cost;
 }
 
+template <typename T>
+int count_items_with_name(const char* name, const T* item_array, int num_items)
+{
+	Assert(name != nullptr && item_array != nullptr);
```

Comment:
This one should be changed to an Assertion so that we can know which of these would be nullptr when triggered.

Reply:
Okay

Reply:
Perhaps... but then I thought of another option.  See latest commit, which just returns 0 or -1 as appropriate.